app/javascript/client/product/modals/ProductModal.tsx

```diff
@@ -19,14 +18,29 @@ const CloseModalButton = styled.button`
     height: 50px;
     cursor: pointer;
 `
+interface ProductModalProps {}
 
-const ProductModal = () => {
+const ProductModal: FC<ProductModalProps> = () => {
 
-    const {modalIsShowing, openModal, closeModal} = useContext(ModalContext);
+    const { selectedProduct, closeModal} = useContext(ModalContext);
+
+    if (!selectedProduct.imageUrl) return null;
+
+    const imageStyles: CSS.Properties = {
```

Comment:
I think it's fine to leave these as inline styles for now but just wondering why / if not use Styled Components instead? I assume it didn't work for some reason? I say we try to debug if so